package.json

```diff
@@ -25,13 +25,11 @@
     "exports": "eyeglass-exports.js"
   },
   "scripts": {
-    "test": "npm run --silent test-libsass && npm run --silent test-ruby",
+    "test": "npm run test-libsass && npm run test-ruby",
```

Comment:
I didn't really know what it was doing, and I wasn't seeing useful errors, so I thought that might help. I could certainly be wrong. :)